### PR TITLE
Added/updated AWS EKS and Azure EKS related ReadMe Files

### PR DESCRIPTION
**Pull request by sundharakanth**

### CODE REVIEW

**Comment by sundharakanth** [comment]
Hi @GuentherJulian and @sujith-mn - As discussed, please help me with this review and merge. Thank you.

File: k8s/aws-k8s-steps.adoc [hunk 1]
```diff
@@ -0,0 +1,62 @@
+== Kubernetes Deployment in AWS EKS (Elastic Kubernetes Service)
+
+=== Pre-requisites
```
Comment (GuentherJulian) [inline k8s/aws-k8s-steps.adoc:3]: Maybe we can also add here that the AWS CLI (and Azure CLI in the other file) needs to be installed and add a link to the corresponding installation guide?

File: k8s/aws-k8s-steps.adoc [hunk 2]
```diff
@@ -0,0 +1,62 @@
+== Kubernetes Deployment in AWS EKS (Elastic Kubernetes Service)
```
Comment (GuentherJulian) [inline k8s/aws-k8s-steps.adoc:1]: Wouldn't it make more sense to place these two documentations in the `documentation` folder instead of `k8s`?

Comment (sundharakanth) [reply k8s/aws-k8s-steps.adoc]: Yes, right. Done.

**Review by GuentherJulian — APPROVED** [review]

**Pull request merged**